scripts/verdaccio.sh

```diff
@@ -161,6 +161,12 @@ then
   yarn build
   mv dist ../../$verdaccio_path/s2-webpack-5-example
 
+  echo 'test icon builder'
+  cd ../../examples/s2-webpack-5-example
+  mkdir icon-test
```

Comment:
Isn't this still technically testing within our repo? i.e. `../../node_modules` will exist (outside examples folder). So even if the dependencies were missing they'd still get picked up here?

Reply:
Right, this is the reason why running the icon builder inside our repo works (we have svgo as part of a different dependency) but not outside our repo if I understand the original issue correctly